docs/syntax.rst

```diff
@@ -74,10 +102,9 @@ Interpolation and External Arguments
         "emailCount": 5
     }
 
-In FTL strings may use special syntax to incorporate small pieces of 
-programmable interface. Those pieces are denoted with curly braces ``{`` and 
-``}`` and are called placeables.  The most common use case for a placeable is 
-to put an external argument, provided by the developer, into the string.
+
+The most common use case for a placeable is to put an external argument,
```

Comment:
Since we've just mentioned entity references:

    `Another common use case...`